src/documentation/pages/components-wip/forms/radio-button/examples/_error.vue

```diff
@@ -32,33 +28,44 @@ import { Vue } from 'vue-facing-decorator';
     return {
       exampleTabs: [
         {
-          disabled: true,
+          active: true,
           id: 'webcomponent',
           label: 'Web Component',
         },
         {
-          active: true,
+          disabled: false,
           id: 'htmlblueprint',
           label: 'HTML Blueprint',
         },
       ],
       codeSnippets: {
-        webcomponent: ``,
+        webcomponent: `<fieldset>
+  <legend class="chi-label">
+    Select an option
+    <abbr class="chi-label__required" aria-label="Required field">*</abbr>
+  </legend>
+  <div class="chi-form__item>
+    <chi-radio-button id="example__error_opt_1" label="Option 1" name="radios" state="danger" required></chi-radio-button>
+  </div>
+  <div class="chi-form__item>
+    <chi-radio-button id="example__error_opt_2" label="Option 2" name="radios" state="danger" required></chi-radio-button>
+  </div>
+</fieldset>`,
```

Comment:
Missing snippet here?

```js
<div class="chi-label -status -danger">
  <i class="chi-icon icon-circle-warning" aria-hidden="true"></i>
  Please select an option
</div>
```

Reply:
It looks ok. I don't understand what you mean.